### ISSUE TITLE
Incorrect request when trying to import more than 100 xliff files

### ISSUE THREAD

**Issue by osmers** [digitalfabrik/integreat-cms#3166]
### Describe the Bug
Server responds with 400 error for incorrect request when trying to upload multiple xliffs at once.

### Steps to Reproduce

1. Go to https://admin.integreat-app.de/testumgebung/pages/de/
2. Download xliffs for any language (needs to be more than 100)
3. Extract .zip file content into a new folder
4. Go to pages and scroll down to XLIFF import section
5. Click select files and browse to folder with extracted xliff files
6. Select more than 100 xliff files (e.g. 101)
7. Click Import
8. See error

### Expected Behavior
Xliffs get uploaded and the preview/confirmation screen is shown.

### Actual Behavior
400 error

### Additional Information
![image](https://github.com/user-attachments/assets/1de3125d-9095-4428-9a6f-a5c790d64720)

**Comment by MizukiTemma** [comment]
@juliankehne 
Could you test whether this bug can be reproduced?

I failed both on the test/live system and the local environment.

**Comment by JoeyStk** [comment]
Added `Question` label because it's unclear if this is can be reproduced

**Comment by juliankehne** [comment]
@MizukiTemma @JoeyStk Yes, I was able to reproduce it. I updated the steps to reproduce and added some details I found. Hope it helps :)

**Comment by JoeyStk** [comment]
FYI (probably for @juliankehne): This issue is not fixed, but there is a warning shown now
